src/krate.rs

```diff
@@ -174,8 +175,9 @@ impl Crate {
         })));
 
         try!(conn.execute("INSERT INTO crate_owners
-                           (crate_id, user_id, created_at, updated_at, deleted)
-                           VALUES ($1, $2, $3, $3, FALSE)",
+                           (crate_id, owner_id, created_by, created_at,
```

Comment:
Note: I added the created_by field here while debugging a test failure cause by something else. Not sure why it wasn't being set and whether it should be set.